src/uwtools/tests/test_cli.py

```diff
@@ -385,39 +394,13 @@ def test__dispatch_template_render_yaml():
     )
 
 
-def test__dispatch_template_translate():
-    args = {
-        STR.infile: 1,
-        STR.outfile: 2,
-        STR.dryrun: 3,
-    }
-    with patch.object(
-        uwtools.api.template, "_convert_atparse_to_jinja2"
-    ) as _convert_atparse_to_jinja2:
-        cli._dispatch_template_translate(args)
-    _convert_atparse_to_jinja2.assert_called_once_with(input_file=1, output_file=2, dry_run=3)
-
-
-def test__dispatch_template_translate_no_optional():
-    args = {
-        STR.dryrun: False,
-        STR.infile: None,
-        STR.outfile: None,
-    }
-    with patch.object(
-        uwtools.api.template, "_convert_atparse_to_jinja2"
-    ) as _convert_atparse_to_jinja2:
-        cli._dispatch_template_translate(args)
-    _convert_atparse_to_jinja2.assert_called_once_with(
-        input_file=None, output_file=None, dry_run=False
-    )
-
-
```

Comment:
I'm confused about why these are removed here.

Reply:
Hm, this could be a conflict resolution error, I'll look into it

Reply:
Yep, something went weird there - fixing it!